docker-compose-base.yaml

```diff
@@ -204,7 +204,7 @@ services:
        - ./db2/db2_data:/db2_data
 
   localstack:
-    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
+    container_name: "${LOCALSTACK_DOCKER_NAME:-localstack-main}"
```

Comment:
The default container name for LocalStack is `localstack-main` from [v3](https://github.com/localstack/localstack/releases/tag/v3.0.0 ).